.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,16 @@
+name: pre-commit
```

Comment:
@SebastianSchildt is this okay from an open source legal perspective? Tp run automatically run a code formatter before every commit? We had the dicussion regarding auto generation of licenses too.

Reply:
It actually does not fix anything when run as part of CI. See the dummy PR in https://github.com/erikbosch/kuksa.val.feeders/pull/2 where I have added a blank in a README. The check will fail, it will not fix it.

This can be compared to if you run it locally. There it will fix the file, but it will not do a `git add` and it will not do a `git commit --amend`. So also when running locally you must "accept" the changes (by git add) and then try `git commit` again.  

```
erik@debian3:~/kuksa.val.feeders$ pre-commit run --file README.md 
Trim Trailing Whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing README.md

Fix End of Files.........................................................Passed
Check Yaml...........................................(no files to check)Skipped
Check for added large files..............................................Passed
flake8...............................................(no files to check)Skipped
```

Reply:
Ok, it read like it would fix it. But then also fine.